docs/en/deployment.md

```diff
@@ -203,9 +211,11 @@ Before you launch the solution, review the architecture, supported regions, and
 
     Parameter | Default | Description 
     ---|---|---
-    QuickSightRoleName | <Requires input\> | QuickSight Service Role name, which can be obtained from [Security & permissions](https://us-east-1.quicksight.aws.amazon.com/sn/admin?#aws). 
-    QuickSightUser | <Requires input\> | QuickSight Username, which can be obtained from [Manage users](https://us-east-1.quicksight.aws.amazon.com/sn/admin?#users).                 |
-
+    Deploy Batch Evaluation | no | Choose yes to deploy **Batch Evaluation** part, no to skip |
+    Deploy Visualization | no | Choose yes to deploy **Visualization** part, no to skip |
```

Comment:
what about only choosing `visualization` without `batch evaluation`? does it work or not? what's the except of the customers?

Reply:
it works. customers can deploy batch evaluation later.

Reply:
I'm confused for this user experience. How to use `visualization` part without `batch evaluation`?

Reply:
user cannot use visualization without batch evaluation. but user can choose deploy batch evaluation later.